components/youtube_video/youtube_video.jsx

```diff
@@ -15,6 +15,7 @@ export default class YoutubeVideo extends React.PureComponent {
         currentChannelId: PropTypes.string.isRequired,
         link: PropTypes.string.isRequired,
         show: PropTypes.bool.isRequired,
+        googleDeveloperKey: PropTypes.stringt,
```

Comment:
`stringt` looks like a typo

Reply:
Thanks, @grundleborg! Looks like I missed the console warning when testing this component -- fixed!